### PR TITLE
Zero-shot classification tutorial

### PR DESCRIPTION
**Pull request by vwinland**
adding zero-shot classifcation jupyter notebook to repo

### CODE REVIEW

**Review by aspleenic — APPROVED** [review]
lgtm

File: generative-ai/zero-shot-classification.ipynb [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+    "\n",
+    "Researchers have been experimenting with machine learning models and deep learning algorithms for classification tasks since the 50’s. [The Perceptron](https://courses.cs.washington.edu/courses/cse490h1/19wi/exhibit/machine-learning-1.html#:~:text=Finally%2C%20the%20first%20neural%20network,(History%20of%20the%20Perceptron)) is an early classification model that uses a decision-boundary to classify data into different groups. Most ML/DL [methods](https://www.researchgate.net/publication/359103439_A_Zero-Shot_Learning_Approach_to_Classifying_Requirements_A_Preliminary_Study) rely on supervised learning techniques, and therefore need to be trained using a large amount of task-specific labeled training data. This presents a challenge as the large, annotated datasets required to train these models simply do not exist for every domain. [Some researchers](https://arxiv.org/abs/2312.01044#:~:text=%5B2312.01044%5D%20Large%20Language%20Models%20Are%20Zero%2DShot%20Text%20Classifiers) motivated by these constraints say that LLMs are the way around these data limitations.\n",
```
Comment (joshuajnoble) [inline generative-ai/zero-shot-classification.ipynb:15]: I don't think DL has been defined anywhere?

File: generative-ai/zero-shot-classification.ipynb [hunk 2]
```diff
@@ -0,0 +1,271 @@
...
+    "\n",
+    "- **Foundation models** are multimodal, meaning they are often trained on a myriad of modalities including text, images, and speech. These models are generally versatile and after pretraining can undergo optimization for many different tasks. IBM’s granite models classify data by using a large language model (LLM) trained on a curated dataset of business-relevant information, including legal, financial, and technical domains, which allows it to analyze text and identify patterns to categorize data into specific classes based on the context and semantic meaning within the text. Because of their multimodal capacity, these types of models can handle image and text classification. These models are ideal when you need a broad range of capabilities or want to handle multiple types of data for instance, image or audio classification.\n",
```
Comment (joshuajnoble) [inline generative-ai/zero-shot-classification.ipynb:33]: I think this needs to be upper case Granite Model and a TM

**Review by joshuajnoble — COMMENTED** [review]
Looks good, added two little comments.

File: generative-ai/zero-shot-classification.ipynb (hunk 1 above)
Comment (vwinland) [reply generative-ai/zero-shot-classification.ipynb]: I removed the reference to deep-learning here

File: generative-ai/zero-shot-classification.ipynb (hunk 2 above)
Comment (vwinland) [reply generative-ai/zero-shot-classification.ipynb]: Updated the capitalization and added (TM) to this and one other instance where "IBM Granite Model" is used

**Review by joshuajnoble — APPROVED** [review]

**Pull request merged**